src/app/features/request/request.module.ts

```diff
@@ -4,12 +4,14 @@ import { SharedModule } from '../../shared/shared.module';
 import { RequestComponent } from './request.component';
 import { NewComponent } from './new/new.component';
 import { RequestRoutingModule } from './request-routing.module';
+import { FormsModule } from '@angular/forms';
 
```

Comment:
You can use

import { NgSelectModule } from '@ng-select/ng-select';

and push it in imports 

and use it to select skilles in html